### ISSUE TITLE
Use touch and/or pickup to jump to another step

### ISSUE THREAD

**Issue by abalderas** [WEKIT-ECS/MIRAGE-XR#1210]
In addition to using the gaze during several seconds to jump to another step, it would be interesting to also change step when touching and/or picking up an element.

**Comment by fwild** [comment]
Can you describe what you mean by pickup? Not quite sure what that means... kind of 'onFocus', but not 'onRelease'? What should happen with the picked up thing - should it be released where it was held at the time the event is triggered?

**Comment by abalderas** [comment]
Yes, it is an 'onFocus' event. By picking I mean taking the element, although it can be just touching and the object does not need to be kept in the jump to the next step.

In each step the user have to accomplish some objective, once it is accomplished (touching or taking the element), the user jumps to the next step.

Example for language learning:
Step 1: 
- 10 objects are shown to the user on a table.
- We record the definition in the target language of one of them. 
- The learner recognizes the object from its definition and takes it (or touches it).
- When correct, it jumps to step 2, where a new similar scenario could be proposed but with objects from another context.

**Comment by fwild** [comment]
I think this "touch" trigger can already be done with the "pick and place" augmentation - just that it may be a bit weird to pickup and drop on the same place - it's better to add e.g. a 3d model "plate" and then instruct user to pick the right object and drop it on the plate?

**Comment by fwild** [comment]
Please try - if this works, but is too difficult to author, we can refine the pick and place functionality!

**Comment by abalderas** [comment]
Thanks for your comments! We added several 3D models and several pick and place elementthat had to be dropped to the models. It worked for us, although at the beginning in some cases it was difficult to fit the pick and place with its "base".
